### DISCUSSION TITLE
Need help to get values from 1 woocommerce database into another database (beginner)

### DISCUSSION THREAD

**Discussion by BloemenBezorgen** [Budibase/budibase#7768] [Help]
Hi there, I am new to Budibase and fiddling around to get things working. I connected budibase to my woocommerce database via Mysql data source connection.

I would like to create a list of all my products in my woocommerce shop with the following data. There is one database that contains almost all data that I need: wp_wc_product_meta_lookup:
product_id
sku
price
stock_status

I would like to add the following columns to the wp_wc_product_meta_lookup database:
post_type (product, product_variation)
post_title (name of the product)
Product featured image url

The post_type and post_title can be found on the wp_post database by looking up the product_id (wp_wc_product_meta_lookup) in the id (wp_post). I'm not able to get the values in the wp_wc_product_meta_lookup database though

The url of the featured image can be found in wp_post database as well but the post id of the featured image is stored in the wp_postmeta database where meta_key = _thumbnail_id .

So from wp_postmeta database I need to get the meta_value to find the attachment in wp_posts database
post_id = 32
meta_key = _thumbnail_id 
meta_value = 14381 = id in wp_posts

Can this be done with setting up relationships? I have tried to set up relationships to get the post_tilte in but the values are not displaying in the databases

![Relationship Budibase](https://user-images.githubusercontent.com/82037117/190139627-0d7fd2f5-90db-44f5-a8b7-658a2f947935.png)

![Relationship 2 Budibase](https://user-images.githubusercontent.com/82037117/190140087-de320b3f-5f96-47bc-b8d4-24b1a41b47b9.png)

**Comment by melohagan** [comment]
Hey @BloemenBezorgen 

I'm wondering what the expected relationship is. Seems like it could be Many Posts -> Many Products?

In which case you would select the *Many* relationship type, and use the `wp_wc_product_meta_lookup` table as your **through** table.
